contrib/format-hdf5/src/main/java/org/apache/drill/exec/store/hdf5/HDF5BatchReader.java

```diff
@@ -92,6 +93,20 @@
 
   private static final String LONG_COLUMN_NAME = "long_data";
 
+  private static final String DATA_SIZE_COLUMN_NAME = "data_size";
+
+  private static final String ELEMENT_COUNT_NAME = "element_count";
+
+  private static final String IS_TIMESTAMP_NAME = "is_timestamp";
```

Comment:
@paul-rogers 
I debated adding these.  At the moment, Drill/HDF5 does support reading `TIMESTAMP` columns as `TIMESTAMPS`.  I wasn't aware of intervals as an HDF5 data type so the current implementation doesn't support that. 

I can create a PR to support Intervals and work on it if there is usage/demand for it.  
Do you think I should remove these columns from the metadata view?

Reply:
@cgivre, maybe just add the `extended_type` column (or `hdf5_type`) to tell the user the actual data type, leaving the `data_type` to provide the converted Drill type. This will work whether the code actually supports time/interval conversion or not.

Reply:
@paul-rogers 
Since there is a column reporting the actual HDF5 data type, I removed both these columns.  If there is demand, I'll implement the interval data type.  Timestamp is already implemented.